src/main/scala/rise/core/IsClosedForm.scala

```diff
@@ -6,15 +6,32 @@ import rise.core.traverse._
 import rise.core.types._
 
 object IsClosedForm {
+  case class OrderedSet[T](ordered : Seq[T], unique : Set[T])
```

Comment:
Is `unique` a good name? Writing the predicate `unique(x)` reads like "is x unique?", which seems wrong. What about simply `set`?

Reply:
thanks, done! for consistency I renamed `unique` to `set` and `ordered` to `seq`